adala/runtimes/_litellm.py

```diff
@@ -153,20 +153,32 @@ def _get_usage_dict(usage: Usage, model: str) -> Dict:
     return data
 
 
-class InstructorClientMixin:
+def resolve_litellm_model_and_provider(model_name: str, provider: str):
+    """
+    When using litellm.get_model_info() some models are accessed with their provider prefix
+    while others are not.
+
+    This helper function contains logic which resolves this for supported providers
+    """
+    if "/" in model_name:
```

Comment:
I figure this should also make sure that the first part is the provider? Otherwise if someone uses this on e.g. model_name="meta/llama3.2", provider="vertex_ai", they'd end up with surprising behavior

Reply:
ugh. currently we dont support any models like this but yeah... not sure how to really check this deterministically though

Reply:
wont happen actually. this `model_name` comes from LSE where we always prefix it with `vertex_ai/` for example